hrjobcontract/tests/phpunit/CRM/Hrjobcontract/Import/Parser/ApiTest.php

```diff
@@ -47,6 +53,22 @@ function setUp() {
     ];
   }
 
+  function createPayScale() {
```

Comment:
It's by class, but it actually happens at the start, on setup.  I've never been asked to do a teardown before, but maybe we should check with @davialexandre ?